From f1b982ec03642f5c9a8f2313a17ec39c81ebb397 Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Sat, 6 Feb 2016 03:32:01 +0100
Subject: [PATCH] frameworks/base/packages/SystemUI/AndroidManifest.xml:343:
 error: Error: No resource found that matches the given name (at 'label' with
 value '@string/mland').

Change-Id: Ic86c9866e1744707f5aa85b412a4c06be2430897
---
 packages/SystemUI/AndroidManifest.xml | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/packages/SystemUI/AndroidManifest.xml b/packages/SystemUI/AndroidManifest.xml
index bb2e74a..f455469 100644
--- a/packages/SystemUI/AndroidManifest.xml
+++ b/packages/SystemUI/AndroidManifest.xml
@@ -340,23 +340,6 @@
             </intent-filter>
         </activity>
 
-        <activity android:name=".egg.MLandActivity"
-                  android:theme="@android:style/Theme.Material.NoActionBar"
-                  android:exported="true"
-                  android:icon="@drawable/icon"
-                  android:label="@string/mland"
-                  android:hardwareAccelerated="true"
-                  android:launchMode="singleInstance"
-                  android:screenOrientation="locked"
-                  android:process=":sweetsweetdesserts"
-                  android:excludeFromRecents="true">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN"/>
-                <category android:name="android.intent.category.DEFAULT" />
-                <category android:name="com.android.internal.category.PLATLOGO" />
-            </intent-filter>
-        </activity>
-
         <!-- a gallery of delicious treats -->
         <service
             android:name=".DessertCaseDream"
-- 
1.9.1

